dcpy/utils/s3.py

```diff
@@ -92,9 +92,11 @@ def _progress():
 
 def client() -> S3Client:
     """Returns a client for S3."""
+    aws_s3_endpoint = os.environ["AWS_S3_ENDPOINT"]
+    # when running tests with mock S3 buckets, endpoint_url can be None but not "" or any fake url
+    endpoint_url = None if not aws_s3_endpoint else aws_s3_endpoint
```

Comment:
Sorry, approved the PR, but took another look, and I'm not sure about this. Is there a way to accommodate this in the test, rather than changing util code?

Reply:
yea this was hard to pick an approach for. I'll change it and use a test to show how `botocore` handles the input and throws exceptions

Reply:
@alexrichey I've revised https://github.com/NYCPlanning/data-engineering/pull/510/commits/8343aadbb6229479fcb53f03d51e23e63e0e0fdb